mne/io/edf/edf.py

```diff
@@ -77,6 +77,9 @@ class RawEDF(BaseRaw):
 
     It is also possible to retrieve system codes, but no particular effort has
     been made to decode these in MNE.
+    For GDF files, the stimulus channel is constructed from the events in the
+    header. You should use keyword ``stim_channel=-1`` to add it at the end of
+    the channel list.
```

Comment:
I don't like `stim_channel=-1`, because under Python indexing this should be equivalent to `stim_channel=len(channels) - 1` but it's not. Better to use a keyword like `stim_channel='append'` or something to make it clear you're appending a new channel, not converting an existing channel (i.e., what is done for any other integer value).